src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -954,7 +955,7 @@ private RuntimeExecutionResults ProcessCore()
                         new ProcessingError(
```

Comment:
>ProcessingError [](http://example.com/codeflow?start=28&length=15)

should ProcessingError contain the DataSourceGroup now? the processing mode might be relevant to the error?

Reply:
I thought about that, but wanted to minimize the changes since this PR is already bigger than expected. Happy to open a new work item though 😄

Reply:
whichever you prefer

Reply:
Made #216